### PR TITLE
Update CHS translation to 0.5.1

### PR DESCRIPTION
**Pull request by WuzgXY-GitHub**

### CODE REVIEW

**Comment by vectorwing** [comment]
Thanks for the update!

As for the changes you suggested for the en_us lang - "other" was intentional in this case. The Skillet can be operated from either hand: your ingredient can be on your main-hand, so long as you're not currently able to consume it.

File: src/main/resources/assets/farmersdelight/lang/en_us.json [hunk 1]
```diff
@@ -249,7 +249,7 @@
   "farmersdelight.block.cutting_board.invalid_item": "This doesn't seem cuttable...",
   "farmersdelight.block.cutting_board.invalid_tool": "Maybe with a different tool...",
   "farmersdelight.block.skillet.invalid_item": "This can't be cooked...",
-  "farmersdelight.item.skillet.how_to_cook": "Use your other hand to hold ingredients!",
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/en_us.json]: The use of "other" was intended in this case, as the Skillet can be operated from either hand. Can you please revert the change?

**Review by vectorwing — CHANGES_REQUESTED** [review]

**Comment by WuzgXY-GitHub** [comment]
Oh yeah, I tested it in creative mode, and foods are always edible in this mode. This is completely due to my carelessness, sorry for that.
~~I'm currently far away from my PC, and I forgot the password (which is stored in my PC browser) to log into my GitHub account, could you please edit the file by yourself? I think it is maintainer-editable. Sorry again, for inconvenience I brought.~~
I have reverted the change and updated the Chinese language file.

**Comment by vectorwing** [comment]
Sorry for the really long delay! I got wrapped up into a break and forgot to reply to you. :(

I was going to say it could be done on my end, but thanks for taking the time to revert it. I'll go ahead with the merge. Thanks for the update once again! 👍

**Review by vectorwing — APPROVED** [review]

**Pull request merged**